e2e/e2e-windows-scenario.sh

```diff
@@ -45,6 +44,36 @@ collect-logs() {
     echo "collect cse logs done"
 }
 
+DEPLOYMENT_VMSS_NAME="$(mktemp -u winXXXXX | tr '[:upper:]' '[:lower:]')"
+export DEPLOYMENT_VMSS_NAME
+
+# download azcopy
+wget https://aka.ms/downloadazcopy-v10-linux
+tar -xvf downloadazcopy-v10-linux
+
+# zip and upload cse package
+cd ../staging/cse/windows
+zip -r ../../../e2e/${DEPLOYMENT_VMSS_NAME}-aks-windows-cse-scripts.zip ./* -x ./*.tests.ps1 -x "*azurecnifunc.tests.suites*" -x README -x provisioningscripts/*.md -x debug/update-scripts.ps1
+echo "zip cse packages done"
+
+set +x
+expiryTime=$(date --date="2 day" +%Y-%m-%d)
+token=$(az storage container generate-sas --account-name $STORAGE_ACCOUNT_NAME --account-key $MAPPED_ACCOUNT_KEY --permissions 'rwacdl' --expiry $expiryTime --name $STORAGE_PACKAGE_CONTAINER)
+tokenWithoutQuote=${token//\"}
+
+csePackageURL="https://${STORAGE_ACCOUNT_NAME}.blob.core.windows.net/${STORAGE_PACKAGE_CONTAINER}/${DEPLOYMENT_VMSS_NAME}-aks-windows-cse-scripts.zip?${tokenWithoutQuote}"
+export csePackageURL
+
+cd ../../../e2e
+
+array=(azcopy_*)
+${array[0]}/azcopy copy ${DEPLOYMENT_VMSS_NAME}-aks-windows-cse-scripts.zip "https://${STORAGE_ACCOUNT_NAME}.blob.core.windows.net/${STORAGE_PACKAGE_CONTAINER}/${DEPLOYMENT_VMSS_NAME}-aks-windows-cse-scripts.zip?${tokenWithoutQuote}"
+set -x
+
+echo "upload cse packages done"
```

Comment:
could we exit immediately if it fails in uploading the test cse package?

Reply:
Sure. I will add it